code/datums/craft/recipes/clothing.dm

```diff
@@ -189,13 +189,13 @@
 
 /datum/craft_recipe/clothing/accessory/saber
 	name = "Scabbard"
-	result = /obj/item/clothing/accessory/holster/saber
+	result = /obj/item/storage/pouch/holster/belt/sheath/improvised
```

Comment:
What's the point of this now, you're turning a normal belt sheath into an improvised without changing its name?
Return it to accessory. THAT is just weird.

Reply:
Forgot to change its name to sheath.